### PR TITLE
Dashboard headline bug

### PR DESCRIPTION
**Pull request by rhKashif**
Fixed headline figure for most reviewed release
- Was incorrectly counting duplicate rows of reviews for the same game
- Dropped duplicate rows for `review_id` and count unique `review_id` now

### CODE REVIEW

File: dashboard/utility_functions.py [hunk 1]
```diff
@@ -522,11 +523,14 @@ def sub_headline_figures(df_releases: DataFrame) -> None:
...
         st.metric("Most Released Genre:", df_releases["genre"].mode()[0])
     with cols[1]:
-        st.metric("Most Reviewed Release:", df_releases["title"].mode()[0])
+        st.metric("Most Reviewed Release:", df_releases.drop_duplicates(
+            subset="review_id", inplace=False)["title"].mode()[0])
     with cols[2]:
         st.metric("Most Compatible Platform",
                   compatibility_df["platform"].max().capitalize())
 
+    print()
```
Comment (gawsalya) [inline dashboard/utility_functions.py]: do we need this?

**Review by gawsalya — APPROVED** [review]
looks good!

**Review by DMcCallion — APPROVED** [review]
Other than that print statement, looks good

File: dashboard/utility_functions.py [hunk 2]
```diff
@@ -522,11 +523,14 @@ def sub_headline_figures(df_releases: DataFrame) -> None:
     with cols[0]:
         st.metric("Most Released Genre:", df_releases["genre"].mode()[0])
     with cols[1]:
-        st.metric("Most Reviewed Release:", df_releases["title"].mode()[0])
+        st.metric("Most Reviewed Release:", df_releases.drop_duplicates(
+            subset="review_id", inplace=False)["title"].mode()[0])
```
Comment (angelikavilde) [inline dashboard/utility_functions.py]: inplace=False is automatically selected

**Pull request merged**